app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt

```diff
@@ -161,6 +160,27 @@ class PasswordFragment : Fragment() {
         }
     }
 
+    /**
+     * fuzzy matches the filter against the given string
+     *
+     * based on https://www.forrestthewoods.com/blog/reverse_engineering_sublime_texts_fuzzy_match/
+     *
+     * @param filter the filter to apply
+     * @param str the string to filter against
+     *
+     * @return true if the filter fuzzymatches the string
+     */
+    private fun fuzzyMatch(filter: String, str: String): Boolean {
+        var i = 0
+        var j = 0
+        while (i < filter.length && j < str.length) {
+            if (filter[i].isWhitespace() || filter[i].toLowerCase() == str[j].toLowerCase())
```

Comment:
Please retain the `Locale.ROOT` parameter in `toLowerCase` calls or change it to `Locale.getDefault()`